src/vcpkg/base/unicode.cpp

```diff
@@ -190,15 +191,15 @@ namespace vcpkg::Unicode
         {
             switch (static_cast<utf8_errc>(condition))
             {
-                case utf8_errc::NoError: return "no error";
-                case utf8_errc::InvalidCodeUnit: return "invalid code unit";
-                case utf8_errc::InvalidCodePoint: return "invalid code point (>0x10FFFF)";
-                case utf8_errc::PairedSurrogates:
-                    return "trailing surrogate following leading surrogate (paired surrogates are invalid)";
-                case utf8_errc::UnexpectedContinue: return "found continue code unit in start position";
-                case utf8_errc::UnexpectedStart: return "found start code unit in continue position";
-                case utf8_errc::UnexpectedEof: return "found end of string in middle of code point";
-                default: return "error code out of range";
+                case utf8_errc::NoError: return msg::format(msgNoError).extract_data();
+                case utf8_errc::InvalidCodeUnit: return msg::format(msgInvalidCodeUnit).extract_data();
+                case utf8_errc::InvalidCodePoint:
+                    return msg::format(msgInvalidCodePoint).append_raw(" (>0x10FFFF)").extract_data();
+                case utf8_errc::PairedSurrogates: return msg::format(msgPairedSurrogatesAreInvalid).extract_data();
+                case utf8_errc::UnexpectedContinue: return msg::format(msgContinueCodeUnitInStart).extract_data();
+                case utf8_errc::UnexpectedStart: return msg::format(msgStartCodeUnitInContinue).extract_data();
+                case utf8_errc::UnexpectedEof: return msg::format(msgEndOfStringInCodeUnit).extract_data();
+                default: return msg::format(msgErrorCodeOutOfRange).extract_data();
```

Comment:
```suggestion
                default: Checks::unreachable(VCPKG_LINE_INFO);
```

and delete ErrorCodeOutOfRange